README.md

```diff
@@ -38,7 +36,7 @@ For browser usage, you can also directly rely on unpkg :
 ```js
 const DashJS = require("dash");
 
-const sdk = new DashJS.SDK({
+const sdk = new DashJS.Client({
```

Comment:
I thought we agreed on renaming `DashJS` to `Dash`

Reply:
Documentation updated to reflect that (pr #39 )